### ISSUE TITLE
[Trainer] Any PNGs in bucket cause failure to train

### ISSUE THREAD

**Issue by James-Knipe** [cloud-annotations/training#127]
Please advise on "Training was canceled or failed.” This error consistently occurs after each apparently successful training.

Extra Info

Bucket location: eu-gb. 
145 images.
2 labels
407 of label 1
67 of label 2
One of the images has 29 boxes. Most have 1-5. others 5-10.

TERMINAL INFO
Watson Machine Learning Credentials
instance_id: (xxxxxxx) 
api_key: (xxxxxxxs) 
url: (https://eu-gb.ml.cloud.ibm.com) 

Cloud Object Storage Credentials
access_key_id: (xxxxxx) 
secret_access_key: (xxxxxxx) 
region: (eu-gb) 

Buckets
training data bucket: checkerseu
output bucket: checkerseuresult

Training Params
gpu: (k80) 
steps: (500) 

project name: (countcheckerseu) 

About to write to /Users/jamesknipe/config.yaml

name: countcheckerseu
buckets:
  training: checkerseu
  output: checkerseuresult
trainingParams:
  gpu: k80
  steps: '500'

Is this ok? (yes) 
James-Knipes-Computer:~ jamesknipe$ cacli train
(Using settings from config.yaml)
success Training run submitted.

Model ID:
┌────────────────┐
│ model-khkwrbdw │
└────────────────┘

Would you like to monitor progress? (yes) 

success Model files saved to bucket.
✨ Done.
James-Knipes-Computer:~ jamesknipe$ cacli download model-khkwrbdw
Training was canceled or failed.

**Comment by James-Knipe** [comment]
I noticed this in the log:

ValueError: Image format not JPEG 
101 training and 44 validation examples.
CACLI-TRAINING-FAILED
Training exited with error code 1
Failed: learner_exit_code: 1



.. And there were PNGs in the bucket. 
A subsequent training with only jpegs allows download of model.

Suggest mentioning this in the "Preparing Training Data" section of the tutorial.

**Comment by bourdakos1** [comment]
Hmm I wasn’t aware PNGs weren’t supported, either way I believe the cloud annotations tool stores all the images as jpgs, did you upload images to your bucket via another method? Also, it shouldn’t fail to train if an invalid file is present, if so that’s a bug that needs to be fixed

**Comment by bourdakos1** [comment]
Also I don’t recommend posting your credentials in a public forum

**Comment by James-Knipe** [comment]
thought so eek: can I delete the post?

… Images were upload by drag n drop straight after  creating abucket on IBM cloud

**Comment by James-Knipe** [comment]
added 1 .png image to bucket - training failed where previously it had passed. 
Tried 2 times - with and without extension. Both failed

**Comment by bourdakos1** [comment]
Okay, good to know, and i recommend deleting your credentials and generate new ones

**Comment by bourdakos1** [comment]
Trainer raises the following error, but was supposed to be caught in a try catch that wasn't wrapping the right code. I'll push a fix
```
if image.format != 'JPEG':
    raise ValueError('Image format not JPEG')
```

**Comment by bourdakos1** [comment]
should be fixed in cacli v1.1.12
